lib/cucumber/rake/task.rb

```diff
@@ -113,7 +113,15 @@ def run
       attr_reader :cucumber_opts
 
       def cucumber_opts=(opts) # :nodoc:
-        @cucumber_opts = String == opts.class ? opts.split(' ') : opts
+        unless String == opts.class
+          @cucumber_opts = opts
+          return
+        end
+
+        @cucumber_opts = opts.split(' ')
+        return if @cucumber_opts.length <= 1
+
+        $stderr.puts 'WARNING: consider using an array rather than a space-delimited string with cucumber_opts to avoid undesired behavior.'
```

Comment:
Generic question. For all of our warnings we emit, do we use stderr or do we use some custom logger (I can't remember). i.e. for the deprecation stuff is this also the way we do it?

Reply:
Yes, we use $stderr with all warnings (there are not that many actually :p)

Reply:
With the deprecation helper, we default to $stderr, but there is the possibility to override it with a stream